<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.9.0 on win32)" generated="20220524 07:59:10.830" rpa="false">
<suite id="s1" name="LogIn" source="C:\Users\parma\PycharmProjects\Robot\LogIn.robot">
<test id="s1-t1" name="LogIn Test">
<kw name="Open Browser">
<arguments>
<arg>http://newtours.demout.com</arg>
<arg>Chrome</arg>
</arguments>
<msg timestamp="20220524 07:59:10.877" level="FAIL">No keyword with name 'Open Browser' found.</msg>
<status status="FAIL" starttime="20220524 07:59:10.877" endtime="20220524 07:59:10.877"></status>
</kw>
<status status="FAIL" starttime="20220524 07:59:10.877" endtime="20220524 07:59:10.877" critical="yes">No keyword with name 'Open Browser' found.</status>
</test>
<status status="FAIL" starttime="20220524 07:59:10.845" endtime="20220524 07:59:10.877"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="LogIn">LogIn</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220524 07:59:10.877" level="ERROR">Error in file 'C:\Users\parma\PycharmProjects\Robot\LogIn.robot' on line 3: Importing test library 'seleniumLibrary' failed: ModuleNotFoundError: No module named 'seleniumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python\Scripts\robot.exe
  c:\python\python39.zip
  c:\python\DLLs
  c:\python\lib
  c:\python
  C:\Users\parma\AppData\Roaming\Python\Python39\site-packages
  c:\python\lib\site-packages
  c:\python\lib\site-packages\win32
  c:\python\lib\site-packages\win32\lib
  c:\python\lib\site-packages\Pythonwin</msg>
</errors>
</robot>
